modules/database/src/ioc/as/asIocRegister.c

```diff
@@ -89,8 +101,14 @@ static void asprulesCallFunc(const iocshArgBuf *args)
 static const iocshArg aspmemArg0 = { "asgname",iocshArgString};
 static const iocshArg aspmemArg1 = { "clients",iocshArgInt};
 static const iocshArg * const aspmemArgs[] = {&aspmemArg0,&aspmemArg1};
-static const iocshFuncDef aspmemFuncDef = {"aspmem",2,aspmemArgs,
-                                           "List members of Access Security Group.\n"};
+static const iocshFuncDef aspmemFuncDef = {
+    "aspmem",2,aspmemArgs,
+    "List members of Access Security Group.\n"
+    "If no Group is specified then print the members for all Groups.\n"
+    "If clients is (0, 1) then Channel Access clients attached to each member\n"
+    "(are not, are) shown\n"
+    "Example: asgname mygroup 1\n",
```

Comment:
Should be

> Example: aspmem mygroup 1

Reply:
Resolved in 7370bba0c841d6564a735bc6d57297a3b7e43e3c